test/fixtures/output/ip_route.out

```diff
@@ -1,3 +1,3 @@
-10.13.32.0/24 dev eth0  proto kernel  scope link  src 10.13.32.195 
-10.163.0.1 dev tun0  proto kernel  scope link  src 10.163.0.6 
-10.163.2.0/28 dev eth1  proto kernel  scope link  src 10.163.2.1 
+10.13.32.0/24 dev eth0  proto kernel  scope link  src 10.13.32.195  metric 600 
+10.163.0.1 dev tun0  proto kernel  scope link  src 10.163.0.6  metric 600 
+10.163.2.0/28 dev eth1  proto kernel  scope link  src 10.163.2.1  metric 600 
```

Comment:
Maybe I could get rid of the `metric` part on the last line. In the virtual machine where I detected the bug, this is the routes table:

```
default via 192.168.122.1 dev ens3  proto static  metric 100 
192.168.122.0/24 dev ens3  proto kernel  scope link  src 192.168.122.153  metric 100
```